templates/vue/src/components/NodeModal.vue

```diff
@@ -257,16 +293,19 @@ export default {
     },
     canPublish() {
       if (this.loading) return false
-      if (this.type === "add") {
+      if (
+        this.type === "add" ||
+        (this.type === "edit" && this.node.status === "draft")
+      ) {
         return (
           Helpers.hasPermission(this.parent, this.type) &&
           (!this.parent || this.parent.status !== "draft")
         )
-      } else if (this.node.status === "draft" && this.type === "edit") {
+      } else if (this.node.status !== "draft" && this.type === "edit") {
         return this.getNeighbours(this.nodeId).some(neighbourId => {
           let neighbour = this.getNode(neighbourId)
           return (
-            neighbour.status !== "draft" && Helpers.hasPermission(neighbour, "add")
+            neighbour.status === "publish" && Helpers.hasPermission(neighbour, "add")
```

Comment:
If a node is published, why would it matter what the status of its neighbours are?

Reply:
Yeah this check doesn't make much sense - removed.